src/style.css

```diff
@@ -40,6 +40,7 @@
   background-color: #95ffcd;
   text-align:center;
 }
-strong {
+/*strong {
```

Comment:
Instead of commenting it out, let's remove it altogether.

Reply:
removed it all together